core/sqf/sql/scripts/install_hadoop_regr_test_env

```diff
@@ -134,12 +134,12 @@ fi
 
 if [ ! -f $MY_LOCAL_SW_DIST/${TPCDS_ZIP} ]; then
   # Right now there is no URL to download this tool automatically
-  echo '*********************** ERROR *************************'
-  echo "Please download the tpcds_kit.zip file from"
-  echo "http://www.tpc.org/tpcds/dsgen-download-request.asp"
-  echo "and place it into a directory pointed to by the MY_LOCAL_SW_DIST"
-  echo "environment variable. Then retry installing. Sorry, this is due"
-  echo "to the TPC wanting your email address."
+  # Please download the tpcds_kit.zip file from"
+  # "http://www.tpc.org/tpcds/dsgen-download-request.asp"
+  # "and place it into a directory pointed to by the MY_LOCAL_SW_DIST"
+  # "environment variable. Then retry installing. Sorry, this is due"
+  # "to the TPC wanting your email address."
+  echo "The testware tpcds_kit.zip does not exist and will not be installed"
   exit 99
 fi
 
```

Comment:
Most of the contributors that build Trafodion code are expecting things to work without change including the running of all regressions.  To do this, a repository has been created that contains everything these contributors need.  However, not all contributors have this luxury.  As Hans pointed out, other contributors, cannot even follow the directions to download this software.  That said, our regression tests require this to be installed - at least our Hive tests.  Let me write a JIRA to on this.  In the meantime, I can display something to the effect that some of the developer regressions will not pass.